change-notes/1.24/analysis-python.md

```diff
@@ -4,18 +4,31 @@ The following changes in version 1.24 affect Python analysis in all applications
 
 ## General improvements
 
-Support for Django version 2.x and 3.x
+- Support for Django version 2.x and 3.x
 
-## New queries
+- Taint tracking now correctly tracks taint in destructuring assignments. For example, if `tainted_list` is a list of tainted tainted elements, then
+    ```python
+    head, *tail = tainted_list
+    ```
+    will result in `tail` being tainted with the same taint as `tainted_list`, and `head` being tainted with the taint of the elements of `tainted_list`.
+
+- A large number of libraries and queries have been moved to the new `Value` API, which should result in more precise results.
+
+- The `Value` API has been extended in various ways:
```

Comment:
In documentation, we try to avoid using the term "API" to talk about the CodeQL libraries since it's open to misinterpretation. Would it make sense to talk about the `Value` interface or library instead?

Reply:
Oh, whoops! We ended up using API the [last time around](https://github.com/Semmle/ql/blob/master/change-notes/1.23/analysis-python.md), so I figured I'd do the same this time. Using "interface" is probably better, though.